doc/third-party/fivetran/setup-guide.md

```diff
@@ -0,0 +1,54 @@
+---
+name: Setup Guide
+title: Materialize Setup Guide
+description: Follow the guide to set up Materialize as a destination.
+---
+
+# Materialize Setup Guide {% badge text="Partner-Built" /%} {% badge text="Private Preview" /%}
+
+Follow the setup guide to connect Materialize to Fivetran.
+
+> NOTE: This destination is [partner-built](/docs/partner-built-program). For any questions related to the Materialize destination and its documentation, contact the [Materialize team](mailto:support@materialize.com).
+
+---
+
+## Prerequisites
+
+To connect Materialize to Fivetran, you need the following:
+
+- A Materialize account. If you already have one - great! If not, [sign up for a playground account](https://materialize.com/register/) first.
+- A Fivetran account with [permission to add destinations](/docs/using-fivetran/fivetran-dashboard/account-management/role-based-access-control#legacyandnewrbacmodel).
+
+---
+
+## Setup instructions
+
+### <span class="step-item"> Get Materialize credentials </span>
+
+The user you're using to connect to Fivetran must have [`CREATE`](https://materialize.com/docs/manage/access-control/rbac/#privileges) privileges on the target database in Materialize.
```

Comment:
Ideally we'd recommend users create a service account in Materialize for this vs using their personal account, but I just looked in our docs and we don't have anything on creating service accounts.

@morsapaes is that something we should put in the Materialize docs, a best practice guide around creating dedicated service accounts for integrations?

That shouldn't block these docs, since it applies to all integrations and we should have a dedicated landing page on it first. Don't want to slow people down in getting set up for the first time.

Reply:
That makes sense to me! I'm going to merge as-is and send them over to Fivetran to unblock them, we can always update if need be!